sst/services/functions/persist.ts

```diff
@@ -1,11 +1,12 @@
+import { Table } from '@serverless-stack/node/table';
 import { APIGatewayProxyHandlerV2 } from 'aws-lambda';
 import dynamodb from 'aws-sdk/clients/dynamodb';
 const documentClient = new dynamodb.DocumentClient();
 
 export const handler: APIGatewayProxyHandlerV2 = async (event) => {
   const item = JSON.parse(event.body || '{}');
   const params = {
-    TableName: process.env.TABLE_NAME as string,
+    TableName: Table.table.tableName,
```

Comment:
Since `table` was bound to this function, we can reference `table.tableName` in a type safe way.